examples/gadgets/simple/trace_open/README.md

```diff
@@ -0,0 +1,31 @@
+# `trace_open`
+
+This example shows how to run the `trace_open` gadget and print the events it
+captures to the terminal in json format.
+
+### How to compile
+
+```bash
+$ go build .
+```
+
+### How to run
+
+The compiled binary doesn't need any parameters, just run it with root permissions:
+
+```bash
+$ sudo ./trace_open
+```
+
+In another terminal, execute some processes (as root because others are filtered out)
+
+```bash
+$ sudo cat /dev/null
+```
+
+Those will printed in the gadget's terminal:
+
+```bash
+$ sudo ./trace_open
+{"args":"/usr/bin/cat","args_count":2,"args_size":23,"comm":"cat","gid":0,"loginuid":1001,"mntns_id":4026531841,"pid":9999,"ppid":98133,"retval":0,"sessionid":3,"timestamp":17145168898503,"uid":0,"upper_layer":false}
```

Comment:
Mmmmm what am I doing wrong? I have different fields:

```bash
{
  "comm": "iio-sensor-prox",
  "err": 0,
  "fd": 8,
  "flags": 2048,
  "fname": "/dev/iio:device0",
  "gid": 0,
  "mntns_id": 4026532919,
  "mode": 0,
  "pid": 1358,
  "timestamp": 55999871673151,
  "uid": 0
}
```

Reply:
I updated it with the latest output. Probably something changed in between.